docs/development.md

```diff
@@ -111,5 +113,51 @@ If you modify any files in `pkg/apis`, you will likely need to regenerate the
 generated clients and other generated files.
 
 1.  Run `./hack/update-codegen.sh` to update the files.
-1.  Add the generated files to your PR, preferably in a separate,
-    generated-only commit so that they are easier to review.
+1.  Add the generated files to your PR, preferably in a separate, generated-only
+    commit so that they are easier to review.
+
+## Nightly releases
+
+The cluster registry has a script, [`hack/release.sh`](../hack/release.sh), that
+is used to build releases and push them for public consumption. This script is
+run nightly by Prow.
+
+### Binaries
+
+Binaries are stored in Google Cloud Storage, in the `crreleases` bucket.
+Currently there are only binaries releases for 64-bit Linux.
+
+To get the latest client library (i.e., `crinit`), run:
+
+```sh
+PACKAGE=client
+LATEST=$(curl https://storage.googleapis.com/crreleases/nightly/latest)
+curl -O http://storage.googleapis.com/crreleases/nightly/$LATEST/clusterregistry-$PACKAGE.tar.gz
```

Comment:
It's not necessary at this pre-alpha stage where everything is expected to change, but in the near future we need to start including platform (linux) and arch (amd64) in the URL path to not break the paths in the URLs.

Reply:
Yes, agreed. Bazel 0.8.0 supports cross compilation, and once that's available, I'll migrate this repo to use the newest version of `rules_go` and update this script to cross compile.